frontend/src/pages/Projects/index.tsx

```diff
@@ -25,7 +25,7 @@ export const Projects = () => {
 
   const handleTableNewButton = async () => {
     await handlePost({ formValues: [], apiUrl: apiUrl as string }).then((response) => {
-      navigate(`/${apiUrl}/${response.id}`);
+      navigate(`/${apiUrl}/${response[0].id}`);
```

Comment:
is this because you are returning back '*' on the new project now, I always cringe when i see an `[0]`, is this the only way to do this, what if this response fails and returns back an error, is this possible, and if so what are the side affects